docs-v2/integrations/all/readwise.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Readwise
+sidebarTitle: Readwise
+provider: readwise
+---
+
+import Overview from "/snippets/overview.mdx";
+import PreBuiltTooling from "/snippets/generated/blandai/PreBuiltTooling.mdx";
+
+import PreBuiltUseCases from "/snippets/generated/blandai/PreBuiltUseCases.mdx"
```

Comment:
@nalanj  what do people need to run to generate those files correctly?

(also it's pointing to blandai instead of readwise)

Reply:
Sorry about that, I think `npm run docs` will generate these snippet files. I have done that in the latest commit.

Reply:
You can also run `npm run docs:generate` to skip running the docs server and just generate the dynamic parts.